### PR TITLE
Country page responsive

### PR DESCRIPTION
**Pull request by hectoruch**
Change:
- Header styles.
- Graphs and sections styles.
- Add summary section.
- Add functions when clicking summary option, the page scroll to the section.

responsive_

### CODE REVIEW

**Comment by hectoruch** [comment]
Add:
- Modal templates.
- Modal functions (open and close).

modal_1

**Comment by hectoruch** [comment]
Add:
- Tree cover loss alerts modal.
- Tree cover loss modal.

Delete:
- Footer on mobile version.

**Comment by hectoruch** [comment]
- Include functions to update graphs with modal data.

treecoverlossalertsmobile

**Comment by hectoruch** [comment]
Add:
- Annual tree cover loss modal functions.

datamodal

**Comment by hectoruch** [comment]
image

File: app/assets/javascripts/countries/views/CountryShowView.js [hunk 1]
```diff
@@ -88,10 +95,15 @@ define([
       this.initFiresAlerts();
       this.initMapCountry();
       this.initStickyMap();
+      this.initSwitchOptions();
 
       this.$el.find('.widgets > .content').removeClass('-loading');
     },
 
+    initAppendBackgroundModal: function() {
+      $('body').append('<div class="background-modal -hidden"></div>');
```
Comment (geriux) [inline app/assets/javascripts/countries/views/CountryShowView.js]: Let's move this to the rails template since we'll always need it there

**Pull request merged**